Source/WebKit/UIProcess/API/haiku/PageClientImplHaiku.h

```diff
@@ -39,6 +39,7 @@ class DrawingAreaProxy;
 class WebViewBase;
 
 class PageClientImpl: public PageClient {
+    WTF_MAKE_FAST_ALLOCATED;
```

Comment:
if I remember correctly (it's been a while since I last looked into this), the classes in UIProcess/API are meant to be used directly by the "UI process", that is, they should be usable outside of WebKit and behave like "normal" classes.

WTF_MAKE_FAST_ALLOCATED may prevent creating them with the normal "new" operator from outside the WebKit tree (I'm not sure if that's really the case, I only vaguely remember hitting this problem, sorry).

It may be OK if the class is created on WebKit side and then passed on to the browser.